AppCreationScripts/Apps.json

```diff
@@ -12,11 +12,11 @@
       "logoutUrl":  "https://localhost:8080/msal4jsample/sign-out",
```

Comment:
This should also be a http URL?

Reply:
Portal won't let you use http in signout URL. Signout functionality still works like this.